app/services/monitor-http/core-count-metrics.ts

```diff
@@ -0,0 +1,11 @@
+import { MonitorChartAggregation, MonitorChartMetrics, MonitorMetricsBase } from "./monitor-metrics-base";
+
+const statesColor = [
+    { state: "corecount", color: "#1C3F95" },
```

Comment:
Should all color values be in CSS rather than hard coded? We are moving towards themes so it would make sense to move all of these values (and all other # values below) into a CSS file.

Reply:
can't really do that here as its for the graph. I am however working on loading themes from a json file so you can use the color variables from javascript and from css

Reply:
Thanks, will remove other related comments